python/lsst/cp/pipe/ptc/cpExtractPtcTask.py

```diff
@@ -168,16 +169,16 @@ class PhotonTransferCurveExtractTask(pipeBase.PipelineTask,
     be validated, and in the future, we may decide to just keep
     one (covariance).
 
-    The measured covariances at a particular time (along with
-    other quantities such as the mean) are stored in a PTC dataset
-    object (``PhotonTransferCurveDataset``), which gets partially
-    filled. The number of partially-filled PTC dataset objects
-    will be less than the number of input exposures, but gen3
+    The measured covariances at a particular time (along with other
+    quantities such as the mean) are stored in a PTC dataset object
+    (``~lsst.ip.isr.PhotonTransferCurveDataset``), which gets
```

Comment:
This string is printed verbatim in the build. Did you mean to use single backticks?